class.jetpack-jitm.php

```diff
@@ -56,490 +55,268 @@ function get_emblem()
-		$jitm_stats_url = Jetpack::build_stats_url( array( 'x_jetpack-jitm' => 'akismet' ) );
-		?>
-		<div class="jp-jitm" data-stats_url="<?php echo esc_url( $jitm_stats_url ); ?>">
-			<a href="#" data-module="akismet" class="dismiss"><span class="genericon genericon-close"></span></a>
-			<?php echo self::get_emblem(); ?>
-			<p class="msg">
-				<?php esc_html_e( "Spam affects your site's legitimacy, protect your site with Akismet.", 'jetpack' ); ?>
-			</p>
-			<p>
-				<a href="<?php echo esc_url( $url ); ?>" target="_blank" title="<?php esc_attr_e( 'Automate Spam Blocking', 'jetpack' ); ?>" data-module="akismet" data-jptracks-name="nudge_click" data-jptracks-prop="jitm-akismet" class="button button-jetpack launch jptracks"><?php esc_html_e( 'Automate Spam Blocking', 'jetpack' ); ?></a>
-			</p>
-		</div>
-		<?php
-		//jitm is being viewed, track it
-		$jetpack = Jetpack::init();
-		$jetpack->stat( 'jitm', 'akismet-viewed-' . JETPACK__VERSION );
-		$jetpack->do_stats( 'server_side' );
+	function get_message_path() {
+		$screen = get_current_screen();
+
+		return 'wp:' . $screen->id . ':' . current_filter();
 	}
 
 	/**
-	 * Display JITM after a post is published prompting user to enable Backups.
-	 *
-	 * @since 3.9.5
-	 */
-	function backups_after_publish_msg() {
-		$normalized_site_url = Jetpack::build_raw_urls( get_home_url() );
-		$url = 'https://jetpack.com/redirect/?source=jitm-backup-publish&site=' . $normalized_site_url;
-		$jitm_stats_url = Jetpack::build_stats_url( array( 'x_jetpack-jitm' => 'vaultpress' ) );
-		?>
-		<div class="jp-jitm" data-track="vaultpress-publish" data-stats_url="<?php echo esc_url( $jitm_stats_url ); ?>">
-			<a href="#" data-module="vaultpress" class="dismiss"><span class="genericon genericon-close"></span></a>
-
-			<?php echo self::get_emblem(); ?>
-
-			<p class="msg">
-				<?php esc_html_e( "Great job! Now let's make sure your hard work is never lost, backup everything with VaultPress.", 'jetpack' ); ?>
-			</p>
-			<p>
-				<a href="<?php echo esc_url( $url ); ?>" target="_blank" title="<?php esc_attr_e( 'Enable Backups', 'jetpack' ); ?>" data-module="vaultpress" data-jptracks-name="nudge_click" data-jptracks-prop="jitm-vault-post" class="button button-jetpack launch jptracks"><?php esc_html_e( 'Enable Backups', 'jetpack' ); ?></a>
-			</p>
-		</div>
-		<?php
-		//jitm is being viewed, track it
-		$jetpack = Jetpack::init();
-		$jetpack->stat( 'jitm', 'vaultpress-publish-viewed-' . JETPACK__VERSION );
-		$jetpack->do_stats( 'server_side' );
+	* Function to enqueue jitm css and js
+	*/
+	function jitm_enqueue_files() {
+		$wp_styles = new WP_Styles();
+		$min = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '' : '.min';
+		wp_enqueue_style( 'jetpack-jitm-css', plugins_url( "css/jetpack-admin-jitm{$min}.css", JETPACK__PLUGIN_FILE ), false, JETPACK__VERSION . '-201243242' );
+		$wp_styles->add_data( 'jetpack-jitm-css', 'rtl', true );
```

Comment:
Unrelated to this specific line, but it looks like we'll need to update the Jetpack icon in that notice for RTL. Notice how the icon has right padding here when it should have left padding. This results in blank space on the right-hand side and no space between text and icon.

![screen shot 2017-05-26 at 10 15 00 am](https://cloud.githubusercontent.com/assets/1126811/26500677/7082b918-41fc-11e7-939b-3433dc5cd7e6.png)